src/components/carousel-with-image/CarouselWithImage.styles.ts

```diff
@@ -17,7 +17,8 @@ export const styles = {
     minHeight: '144px',
     backgroundColor: 'primary.800',
     borderRadius: '6px',
-    p: '16px'
+    p: '16px',
+    m: '0 6px'
```

Comment:
Here we use display: flex, so you can add gap property instead of margin. 
I think it should work.

Reply:
> Here we use display: flex, so you can add gap property instead of margin. I think it should work.

It's actually a great solution to add gap inside a box, but not to add gaps outside it. Svg in the component have "fake" whitespaces that cannot be edited, so basically, i think, adding margins to the description box it's not so bad solution